### PR TITLE
Issuer can submit a credential direct issuance

### PR DESCRIPTION
**Pull request by adrifdez**

### CODE REVIEW

File: ui/src/adapters/api/credentials.ts [hunk 1]
```diff
@@ -450,7 +484,7 @@ export async function createAuthQRCode({
   }
 }
 
-const resultCreatedLinkParser = getStrictParser<ResultCreated<ID>>()(
+const resultCreatedParser = getStrictParser<ResultCreated<ID>>()(
```
Comment (cneuro) [inline ui/src/adapters/api/credentials.ts:487]: Why change this name? Is it to be reused? Currently it doesn't make grammatical sense.

File: ui/src/components/connections/CredentialsTable.tsx [hunk 2]
```diff
@@ -57,6 +60,13 @@ export function CredentialsTable({ userID }: { userID: string }) {
   const [credentialToRevoke, setCredentialToRevoke] = useState<Credential>();
   const [query, setQuery] = useState<string | null>(null);
 
+  const navigate = useNavigate();
+  const navigateIssueDirectly = () =>
```
Comment (cneuro) [inline ui/src/components/connections/CredentialsTable.tsx:64]: To remain consistent, this should probably be called "navigateToDirectIssue".

File: ui/src/adapters/api/credentials.ts (hunk 1 above)
Comment (amonsosanz) [reply ui/src/adapters/api/credentials.ts:487]: It should be `resultCreatedIDParser`

i.e, what we parse (`ResultCreated<ID>`) plus the suffix `parser`.

File: ui/src/adapters/api/connections.ts [hunk 3]
```diff
@@ -67,12 +67,12 @@ export async function getConnection({
...
   credentials,
   env,
-  params: { query },
+  params: { query } = { query: undefined },
   signal,
 }: {
   credentials: boolean;
   env: Env;
-  params: {
+  params?: {
```
[the remark this answers is not in the document]
Comment (adrifdez) [reply ui/src/adapters/api/connections.ts:75]: I prefer this way, it has less code and logics

[the remark this answers is not in the document]
Comment (amonsosanz) [reply ui/src/adapters/api/connections.ts:75]: It's a detail, but if you want my opinion, I'm with Chris here. This feels more natural and readable:

```diff
diff --git a/ui/src/adapters/api/connections.ts b/ui/src/adapters/api/connections.ts
index f80f8a2..c790418 100644
--- a/ui/src/adapters/api/connections.ts
+++ b/ui/src/adapters/api/connections.ts
@@ -67,7 +67,7 @@ export async function getConnection({
 export async function getConnections({
   credentials,
   env,
-  params: { query } = { query: undefined },
+  params,
   signal,
 }: {
   credentials: boolean;
@@ -85,7 +85,7 @@ export async function getConnections({
       },
       method: "GET",
       params: new URLSearchParams({
-        ...(query !== undefined ? { [QUERY_SEARCH_PARAM]: query } : {}),
+        ...(params?.query !== undefined ? { [QUERY_SEARCH_PARAM]: params.query } : {}),
         ...(credentials ? { credentials: "true" } : {}),
       }),
       signal,
```

File: ui/src/components/credentials/IssuanceMethodForm.tsx [hunk 4]
```diff
@@ -7,16 +8,22 @@ import {
...
-  Tag,
   TimePicker,
   Typography,
 } from "antd";
 import dayjs from "dayjs";
-import { useState } from "react";
+import { useCallback, useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import { APIError } from "src/adapters/api";
+import { getConnections } from "src/adapters/api/connections";
```
Comment (amonsosanz) [inline ui/src/components/credentials/IssuanceMethodForm.tsx:19]: extra space

File: ui/src/components/credentials/IssuanceMethodForm.tsx [hunk 5]
```diff
@@ -7,16 +8,22 @@ import {
...
   Row,
   Space,
-  Tag,
   TimePicker,
   Typography,
 } from "antd";
 import dayjs from "dayjs";
-import { useState } from "react";
+import { useCallback, useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
```
Comment (amonsosanz) [inline ui/src/components/credentials/IssuanceMethodForm.tsx:16]: Empty line should be here

**Review by cneuro — APPROVED** [review]

**Review by amonsosanz — APPROVED** [review]

**Pull request merged**